frontend/src/features/archived-items/archived-item-list.ts

```diff
@@ -123,38 +177,127 @@ export class ArchivedItemListItem extends TailwindElement {
+          >
+            <sl-format-bytes
+              class="truncate"
+              value=${this.item.fileSize || 0}
+              display="narrow"
+            ></sl-format-bytes>
+          </sl-tooltip>
         </btrix-table-cell>
+        ${this.listType === "upload"
+          ? nothing
+          : html`
+              <btrix-table-cell>
+                ${isUpload
+                  ? notApplicable
+                  : html`<sl-tooltip
+                      @click=${this.onTooltipClick}
+                      content=${msg(
+                        str`${formatNumber(
+                          this.item.stats?.done ? +this.item.stats.done : 0,
+                        )} crawled, ${formatNumber(this.item.stats?.found ? +this.item.stats.found : 0)} found`,
+                      )}
+                    >
+                      <div class="min-w-4">
+                        ${formatNumber(
+                          this.item.stats?.done ? +this.item.stats.done : 0,
+                          {
+                            notation: "compact",
+                          },
+                        )}
+                      </div>
+                    </sl-tooltip>`}
+              </btrix-table-cell>
+              <btrix-table-cell>
+                ${isUpload
+                  ? notApplicable
+                  : html`<sl-tooltip
+                      @click=${this.onTooltipClick}
+                      content=${this.item.reviewStatus
+                        ? msg(
+                            str`Rated ${this.item.reviewStatus} / ${ReviewStatus.Excellent}`,
+                          )
+                        : msg("No QA review submitted")}
+                    >
+                      <btrix-qa-review-status
+                        .status=${this.item.reviewStatus}
+                      ></btrix-qa-review-status>
+                    </sl-tooltip>`}
+              </btrix-table-cell>
+              <btrix-table-cell>
+                ${isUpload
+                  ? notApplicable
+                  : html`<sl-tooltip
+                      @click=${this.onTooltipClick}
+                      content=${activeQAStats
+                        ? msg("Analysis is currently running")
+                        : lastQAState
+                          ? msg(str`Latest analysis status: ${qaStatus.label}`)
+                          : msg("No analysis runs")}
+                    >
+                      <div
+                        class="flex min-w-4"
+                        style="${qaRunCount
+                          ? ""
+                          : "color: var(--sl-color-neutral-400)"}"
+                      >
+                        ${activeQAStats
+                          ? html`
+                              <sl-progress-ring
+                                value="${activeProgress}"
+                                style="color: ${qaStatus.cssColor}; --size: 20px; --track-width: 1px; --indicator-width: 2px;"
```

Comment:
the microscope icons seem to resolve to about 14px, so maybe that (or something just slightly larger) would be a better since for this indicator?